pyomo/contrib/pyros/separation_problem_methods.py

```diff
@@ -237,29 +242,33 @@ def solve_separation_problem(model_data, config):
     local_solve_time = 0
 
     # List of objective functions
-    objectives_map = model_data.separation_model.util.map_obj_to_constr_names
-    constraint_map_to_master = model_data.separation_model.util.map_constr_list_names_to_original_con_names
+    objectives_map = model_data.separation_model.util.map_obj_to_constr
+    constraint_map_to_master = model_data.separation_model.util.map_new_constraint_list_to_original_con
 
     # Add additional or remaining separation objectives to the dict
     # (those either not assigned an explicit priority or those added by Pyros for ssv bounds)
-    sep_priority_dict = config.separation_priority_order
+    config_sep_priority_dict = config.separation_priority_order
+    actual_sep_priority_dict = ComponentMap()
     for perf_con in model_data.separation_model.util.performance_constraints:
-        if perf_con.name not in sep_priority_dict.keys():
-            sep_priority_dict[perf_con.name] = 0
+        if perf_con.name not in config_sep_priority_dict.keys():
+            actual_sep_priority_dict[perf_con] = 0
+        else:
+            actual_sep_priority_dict[perf_con] = config_sep_priority_dict[perf_con.name]
```

Comment:
FWIW, I was surprised to learn that this is efficient: I used to think of `keys()` as a generator, so would have expected this to either not work or be a linear time operation.  It turns out that this is only slightly less efficient than `perf_con.name not in config_sep_priority_dict`.

That said, this is really just:
```python
actual_sep_priority_dict[perf_con] = config_sep_priority_dict.get(perf_con.name, 0)
```
right?

Reply:
I actually did not know about the dict get() function, I think this is a nicer way to do this...I will make the change.